languages/julia/concepts/iterator-protocol/links.json

```diff
@@ -1 +1,10 @@
-[]
+[
+    {
+        "url": "https://docs.julialang.org/en/v1/manual/interfaces/#man-interface-iteration",
+        "description": "Julia Manual"
+    },
+    {
+        "url": "https://invenia.github.io/blog/2020/11/06/interfacetesting/",
+        "description": "Development with Interface Packages"
+    }
```

Comment:
> This is a good blog post, but without the context of the comment, I think students will think they have to understand all this to use iterators, which obviously they do not.
> 
> I think if this is linked it should be linked from some prose with context.


I added this link again. I think the way it's displayed on the website ([example](http://exercism.lol/tracks/csharp/concepts/floating-point-numbers)) where it's mentioned below the prose that adds context, it will not convey the idea that they have to understand it. It adds a quick shortcut though.

Reply:
I still disagree, but I'm not going to fight on it. Happy for you to merge.

Reply:
For context, this is what it looks like with these changes: http://exercism.lol/tracks/julia/concepts/iterator-protocol